core/src/main/java/org/infinispan/commands/CommandsFactory.java

```diff
@@ -613,17 +613,17 @@ default SingleRpcCommand buildSingleRpcCommand(ReplicableCommand call) {
    BackupNoopCommand buildBackupNoopCommand();
 
    <K, R> ReductionPublisherRequestCommand<K> buildKeyReductionPublisherCommand(boolean parallelStream, DeliveryGuarantee deliveryGuarantee,
-         IntSet segments, Set<K> keys, Set<K> excludedKeys, boolean includeLoader,
+         IntSet segments, Set<K> keys, Set<K> excludedKeys, long explicitFlags,
```

Comment:
What flags do you envision we support now?

Reply:
We probably should document which ones do anything on the Manager interface so we are aware. It was pretty simple to tell we only supported turning off loader or not before.

Reply:
I'm not sure it's useful: the way I see it, ClusterPublisherManager just passes the flags along to the KeySetCommand/CacheSetCommand, and we support whatever flags the interceptors decide to support.
If someone decides an interceptor should support a new flag, they're unlikely to remember to update the ClusterPublisherManager javadoc.